bin/blueoak-server.js

```diff
@@ -3,10 +3,21 @@
  * Copyright (c) 2015-2016 PointSource, LLC.
  * MIT Licensed
  */
+var parseArgs = require('minimist');
 var server = require('../');
 
+// parse arguments
+var argv = parseArgs(process.argv.slice(2));
```

Comment:
You mean like this?
`blueoak-server --mocks mock-service`

Instead of like this?
`./node_modules/blueoak-server/bin/blueoak-server.js --mocks mock-service`

Yes, it works on Node v6.8.1